LLVM_ASAN/Using-Address-Sanitizer-with-a-Short-HIP-Application.md

```diff
@@ -129,13 +123,11 @@ Shadow byte legend (one shadow byte represents 8 application bytes):
   Heap left redzone:       fa
   ...
 ==3141==ABORTING
-
 ```
 
 Running with `m = 100`, `n1 = 10`, `n2 = 10` and `c = 99` should produce a report for an invalid copy.
 
-```
-
+```gdb
```

Comment:
I don't think gdb is a valid code-block language. It might be better to just use text in these cases.